paasta_tools/vitesscluster_tools.py

```diff
@@ -48,8 +564,102 @@ def __init__(
             branch_dict=branch_dict,
         )
 
-    def get_instances(self, with_limit: bool = True) -> int:
-        return self.config_dict.get("replicas", 1)
+    def get_namespace(self) -> str:
+        return self.config_dict.get("namespace", KUBERNETES_NAMESPACE)
+
+    def get_images(self) -> Dict[str, str]:
+        vitess_images = self.config_dict.get(
+            "images", load_system_paasta_config().get_vitess_images()
+        )
+        return {
+            "vtctld": vitess_images["vtctld_image"],
+            "vtadmin": vitess_images["vtadmin_image"],
+            "vtgate": vitess_images["vtgate_image"],
+            "vttablet": vitess_images["vttablet_image"],
+        }
+
+    def get_globalLockserver(self) -> Dict[str, str]:
```

Comment:
```suggestion
    def get_lockserver(self) -> Dict[str, str]:
```
or 
```suggestion
    def get_global_lockserver(self) -> Dict[str, str]:
```
would be more idiomatic

Reply:
https://github.com/Yelp/paasta/pull/3889/commits/6cbc15067cdfc3754ceaf751a8cbe8d8ee749b7b